CHANGELOG.md

```diff
@@ -12,6 +20,7 @@ All notable changes to this project will be documented in this file. The format
 
 ### UNCATEGORIZED PRS; GO LABEL THEM
 
+- \(Maint\) update changelog for 0.8.0 [\#44](https://github.com/puppetlabs/puppetlabs-splunk_hec/pull/44) ([gsparks](https://github.com/gsparks))
```

Comment:
Same will all of the PR's in this section. They all need to be reviewed for proper labeling and then the changelog generator re-run.